tile_pages.py

```diff
@@ -12,10 +12,10 @@
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
 
 import pikepdf
-from pikepdf import _cpphelpers
```

Comment:
I think the _cpphelpers was added to make it play nicely with pyinstaller, so I think I need to keep it

Reply:
Ah, I see. I was a bit puzzled why you import _cpphelpers since it isn't used anywhere in this file, but now it's clear it is needed for the installer. I'll revert that.